c2rust-analyze/src/rewrite/expr/hir_op.rs

```diff
@@ -159,6 +159,23 @@ impl<'a, 'tcx> HirRewriteVisitor<'a, 'tcx> {
                     .unwrap_or_else(|err| panic_sole_location_error(err, "Assign statement"));
                 opt_assign_loc
             }
+            hir::ExprKind::AddrOf(..) => {
+                if ex.span == ex.span.source_callsite() {
+                    // FIXME: this is a hacky proxy for checking if &raw comes from
```

Comment:
this is addressed in https://github.com/immunant/c2rust/pull/893/files#diff-f27494e760407f9ad0a4266c21cf7c2e553c775034319d545a0fba0e58989965L163